RecoLocalCalo/EcalRecAlgos/src/EcalUncalibRecHitTimingCCAlgo.cc

```diff
@@ -152,9 +164,9 @@ float EcalUncalibRecHitTimingCCAlgo::computeCC(const std::vector<float>& samples
   float cc = 0.;
   auto interpolated = interpolatePulse(signalTemplate, time);
   for (int i = exclude; i < int(samples.size() - exclude); ++i) {
-    powerSamples += std::pow(samples[i], 2);
-    powerTemplate += std::pow(interpolated[i], 2);
-    cc += interpolated[i] * samples[i];
+    powerSamples += std::pow(samples[i], 2) * weights[i];
```

Comment:
Not important now since the weights are either 0 or 1, but would it make more sense to have `std::pow(samples[i] * weights[i], 2)` instead when there may be float values for weights?